TLM/TLM/LoadingExtension.cs

```diff
@@ -169,86 +172,74 @@ public class LoadingExtension : LoadingExtensionBase {
-                                    BuildConfig.VersionToString(TrafficManagerMod.GAME_VERSION, false));
-
-                                Log.Error(msg);
-                                Singleton<SimulationManager>.instance.m_ThreadingWrapper.QueueMainThread(
-                                        () => {
-                                            UIView.library
-                                                  .ShowModal<ExceptionPanel>("ExceptionPanel")
-                                                  .SetMessage(
-                                                      "TM:PE has not been updated yet",
-                                                      msg,
-                                                      false);
-                                        });
-                            } else if (isModNewer) {
-                                string msg = string.Format(
-                                    "Traffic Manager: President Edition has been built for game version {0}. " +
-                                    "You are running game version {1}. Some features of TM:PE will not " +
-                                    "work with older game versions. Please let Steam update your game.",
-                                    BuildConfig.VersionToString(TrafficManagerMod.GAME_VERSION, false),
-                                    BuildConfig.applicationVersion);
-
-                                Log.Error(msg);
-                                Singleton<SimulationManager>
-                                    .instance.m_ThreadingWrapper.QueueMainThread(
-                                        () => {
-                                            UIView.library
-                                                  .ShowModal<ExceptionPanel>("ExceptionPanel")
-                                                  .SetMessage(
-                                                      "Your game should be updated",
-                                                      msg,
-                                                      false);
-                                        });
-                            }
-                        }
-
-                        IsGameLoaded = true;
-                        break;
-                    }
-
-                default: {
-                        Log.Info($"OnLevelLoaded: Unsupported game mode {mode}");
-                        return;
-                    }
+            if (BuildConfig.applicationVersion != BuildConfig.VersionToString(
+                    TrafficManagerMod.GAME_VERSION,
+                    false)) {
+                string[] majorVersionElms = BuildConfig.applicationVersion.Split('-');
+                string[] versionElms = majorVersionElms[0].Split('.');
+                uint versionA = Convert.ToUInt32(versionElms[0]);
+                uint versionB = Convert.ToUInt32(versionElms[1]);
+                uint versionC = Convert.ToUInt32(versionElms[2]);
```

Comment:
Would kindly advise a better naming conventions such as
(main).(edition).(repair)

or  (Major version).(Minor version).(Revision number) as suggested here

https://softwareengineering.stackexchange.com/questions/3199/what-version-naming-convention-do-you-use/3225#3225

Reply:
I didn't change this line. you should ignore white spaces.